packages/runtime/container-runtime/src/test/garbageCollection.spec.ts

```diff
@@ -88,6 +92,30 @@ describe("Garbage Collection Tests", () => {
         clock.restore();
     });
 
+    describe("Session expiry is called", () => {
```

Comment:
nit: You can call this "Session expiry" and have the test names more descriptive.